src/llama.cpp

```diff
@@ -6602,7 +6604,8 @@ static void llm_load_vocab(
                 vocab.type_pre = LLAMA_VOCAB_PRE_TYPE_COMMAND_R;
                 vocab.tokenizer_clean_spaces = false;
             } else if (
-                tokenizer_pre == "qwen2") {
+                tokenizer_pre == "qwen2" ||
+                tokenizer_pre == "megrez") {
```

Comment:
This sets `vocab.tokenizer_clean_spaces` to `false`, but 
https://huggingface.co/Infinigence/Megrez-3B-Instruct/blob/main/tokenizer_config.json has `"clean_up_tokenization_spaces": true,`. Is this correct?

Reply:
My bad, not noticed this. Fixed using another if branch.